mindone/utils/load_params.py

```diff
@@ -0,0 +1,136 @@
+import copy
+import os
+from typing import List, Optional, Union
+
+import mindspore as ms
+import mindspore.nn as nn
+from mindspore import Parameter
+from mindspore import log as logger
+
+# from mindspore._checkparam import Validator
+from mindspore.train.serialization import _load_dismatch_prefix_params, _update_param
```

Comment:
if possible, do not use the internal interface, i.e. `_xxx()` from external package, it will be frequently changed without notification.